CMakeLists.txt

```diff
@@ -20,7 +20,20 @@ cmake_minimum_required(VERSION 3.15)
 cmake_policy(SET CMP0069 NEW)  # LTO support
 cmake_policy(SET CMP0091 NEW)  # MSVC runtime support
 
-project(astcenc VERSION 3.0.0)
+project(astcencoder VERSION 3.0.0)
+
+option(UNIVERSAL_BUILD "Build multi-architecture universal binary [only supported on MacOS]" OFF)
```

Comment:
s/MacOS/macOS/

Reply:
Fixed in https://github.com/ARM-software/astc-encoder/pull/266/commits/1ef2b84a31901c36f85479527cfe48d8365546a1